mlflow/store/model_registry/rest_store.py

```diff
@@ -264,6 +265,10 @@ def create_model_version(
 
         """
         proto_tags = [tag.to_proto() for tag in tags or []]
+        if model_id is not None:
+            logged_model = mlflow.get_logged_model(model_id)
+            # models:/<model_id> source is not supported by WSMR
+            source = logged_model.artifact_location
```

Comment:
I thinks we should resolve this in `MlflowClient` or in the fluent API.

Reply:
But should we update this even for UC model registry?